src/cli/index.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-console */
+import { execSync } from 'node:child_process'
+import fs from 'node:fs'
+import path from 'node:path'
+import process from 'node:process'
+import c from 'kleur'
+
+export async function cli() {
+  const cwd = process.cwd()
+
+  console.log(c.dim(`\nSetup eslint config...\n`))
+
+  // Update package.json
+  let pkg: Record<string, any>
+  const pkgPath: string = path.join(cwd, 'package.json')
+
+  try {
+    const pkgContent: string = fs.readFileSync(pkgPath, 'utf-8')
+    pkg = JSON.parse(pkgContent)
+  }
+  catch (error) {
+    console.log(c.red('Could not find -'), c.dim('package.json'))
+    process.exit(1)
+  }
+
+  const eslintLastVersion = await execSync(`npm show eslint version --quite`)
+  const eslintConfigLastVersion = await execSync(`npm show @antfu/eslint-config version --quite`)
+
+  const eslintVersion: string = eslintLastVersion?.toString()?.trim()
+  const eslintConfigVersion: string = eslintConfigLastVersion?.toString()?.trim()
+
+  pkg.type = 'module'
+
+  const pkgDep = ['dependencies', 'devDependencies']
+
+  pkgDep.forEach((depType) => {
+    if (pkg[depType]) {
+      Object.keys(pkg[depType]).forEach((key) => {
+        if (key.includes('eslint') || key.includes('prettier'))
+          delete pkg[depType][key]
+      })
+    }
+  })
+
+  pkg.devDependencies['@antfu/eslint-config'] = eslintConfigVersion
+  pkg.devDependencies.eslint = eslintVersion
+
+  try {
+    fs.writeFileSync(pkgPath, JSON.stringify(pkg, null, 2))
+    console.log(c.cyan('Updated -'), c.dim('package.json'))
+  }
+  catch (error) {
+    console.log(c.red('Failed to update -'), c.dim('package.json'))
+    process.exit(1)
+  }
+  // End update package.json
+
+  // Update eslint files
+  const eslintConfigPath: string = path.join(cwd, 'eslint.config.js')
+  const eslintIgnorePath: string = path.join(cwd, '.eslintignore')
+  let eslintIgnoreLines: string[] = []
+
+  if (fs.existsSync(eslintIgnorePath)) {
+    const eslintIgnoreContent: string = fs.readFileSync(eslintIgnorePath, 'utf-8')
+    eslintIgnoreLines = eslintIgnoreContent.split('\n')
+  }
+
+  const eslintConfigContent = `
+import antfu from '@antfu/eslint-config';
+
+export default antfu({
+  ${eslintIgnoreLines.length
+  ? `ignores: [
+    ${eslintIgnoreLines?.map(line => `'${line}'`).join(',\n')}
+  ],`
+ : ''}
+});`
+
+  const files = fs.readdirSync(cwd)
+  files.forEach((file) => {
+    if (file.includes('eslint') || file.includes('prettier'))
+      fs.unlinkSync(file)
+  })
```

Comment:
This also a bit too aggressive. I think we could print a message to say "you can now remove those files" at the end.